.github/workflows/deployments.yml

```diff
@@ -47,80 +43,17 @@ jobs:
       - name: Install dependencies
         if: steps.cached-poetry-dependencies.outputs.cache-hit != 'true'
         run: make setup
-      - name: Project devnet version
-        id: devnet-version
-        run:
-          echo "DEVNET_VERSION=`poetry run starknet-devnet --version`" >>
-          $GITHUB_OUTPUT
       - name: Deploy
         run: |
-          STARKNET_NETWORK=starknet-devnet poetry run python ./docker/devnet/run_and_deploy.py
-          STARKNET_NETWORK=testnet poetry run python ./scripts/deploy_kakarot.py
-          STARKNET_NETWORK=testnet2 poetry run python ./scripts/deploy_kakarot.py
+          STARKNET_NETWORK=${{ matrix.network }} poetry run python ./scripts/deploy_kakarot.py
```

Comment:
kk, so matrix basically lets you parametrize ci tasks. v cool but weirdly named: https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs

Reply:
Indeed it parametrize and you can parallelize tasks, i.e. building images for multiple platforms 🙂